.pipelines/pipeline.user.windows.yml

```diff
@@ -24,7 +24,7 @@ static_analysis_options:
     files_to_scan:
       - from: 'src\Dependencies\'
```

Comment:
@heyams do you know where this folder comes from? also can we scope this down to the known file that has the issue?

Reply:
this is from onebranch doc.  i can try to narrow it down to that specific dll file.

Reply:
thx